admin_guide/managing_networking.adoc

```diff
@@ -290,7 +290,7 @@ access to `<project B>`. Or restrict application developers from updating from
 [CAUTION]
 ====
 You must have the
-xref:../install_config/configuring_sdn.adoc#install-config-configuring-sdn[*ovs-multitenant* plugin] enabled in order to limit pod access via egress policy.
+xref:../install_config/configuring_sdn.adoc#install-config-configuring-sdn[*ovs-multitenant* or *ovs-networkpolicy* plugin] enabled in order to limit pod access via egress policy.
```

Comment:
@knobunc Would this be better to link to parts of the link where it says to enable the plugins? So:

----
You must have the xref:../install_config/configuring_sdn.adoc#migrating-between-sdn-plugins[*ovs-multitenant* or *ovs-networkpolicy*] plugin enabled in order to limit pod access via egress policy.
----

Alternatively, should this link to the current doc where it says "Enabling NetworkPolicy"[1], although there isn't a section for Multitenent it looks like...

[1] https://docs.openshift.com/container-platform/3.9/admin_guide/managing_networking.html#admin-guide-networking-networkpolicy